codecell/ruby-enumerables
# #Prepare codebase for TSE review

**What does this PR do?**
Gets the codebase ready for TSE review

**How can this manually tested?**
- After cloning this repo to your favorite location
- `cd ruby enumerables`
- in your terminal run `ruby test.rb` to see the result of all the test cases covered
